src/ReportPortal.SpecFlowPlugin/ReportPortalAddin.cs

```diff
@@ -232,8 +230,6 @@ public void AfterScenario()
         }
 
         public delegate void StepStartedHandler(object sender, TestItemStartedEventArgs e);
-        public static event StepStartedHandler BeforeStepStarted;
-        public static event StepStartedHandler AfterStepStarted;
```

Comment:
I would rather implement the events than remove them. It will break backward-compitibility even though the functionality never worked and may not have been used.